library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/UrlTemplate.java

```diff
@@ -115,6 +115,7 @@ public String buildUri(String representationId, long segmentNumber, int bandwidt
    */
   private static int parseTemplate(String template, String[] urlPieces, int[] identifiers,
       String[] identifierFormatTags) {
+    template = template.replace("%00","%0");
```

Comment:
If you're going to do this anywhere, shouldn't it be directly on each `formatTag` down at L148.

Reply:
yeah, maybe I will move  it and then will submit new patch, thx